samples/jwt/uart_logging.conf

```diff
@@ -0,0 +1,16 @@
+# Enable serial printing via UART
```

Comment:
I already asked this in another comment, but does it make much sense to run the sample without log output? Without this overlay, the sample doesn't do anything meaningful. So, I would propose to move these configurations into sample `prj.conf` instead.